packages/table/src/Row.tsx

```diff
@@ -190,17 +210,24 @@ const Row = React.forwardRef(
             React.cloneElement(child, {
               children: <span>{child.props.children}</span>,
               key: `${indexRef.current}-${index}`,
-              disabled: child.props.disabled || disabled,
```

Comment:
Why rm?

Reply:
The `td`/`th` elements don't have a `disabled` attribute. Typing the `isComponentType` call revealed this

Reply:
Ah, good to know 😅